main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
-
-For example, Zoe often uses `offerHandle` to refer to offers. Zoe contracts can use an offer's `offerHandle` as the key for requesting the current allocation of an offer or reallocating the offer's assets.
+A handle is a unique identifier implemented as a JavaScript object. Only its identity 
+is meaningful, so handles do not have properties. Unlike number or string identifiers, 
+handles are unforgeable. This means the only way to know a handle identity is being given 
+an object reference, and no identity can be guessed and no fake identity will succeed. 
+
+## Harden
+A hardened object’s properties cannot be changed, so the only way to interact with a hardened object is through its methods.
+`harden()` is similar to `Object.freeze()` but more powerful. For more about `harden()`, see
+its [section in the JavaScript Distributed Programming Guide](/distributed-programming.md#harden)
+
+## IBC
+The Inter-Blockchain Communication protocol, used to by blockchains to communicate with each other. A short article about IBC
+is available [here](https://www.computerweekly.com/blog/Open-Source-Insider/What-developers-need-to-know-about-inter-blockchain-communication).
+
+## Invitation
+To participate in a contract instance, one must hold an invitation to do so. Contracts often 
+return a creator invitation on their instantiation, in case the contract instantiator wants 
+to immediately participate. Otherwise, the contract instance must create any additional invitations. 
+These, or any invitation held by a party, can be distributed via any means the holder wishes. An
+invitation could be emailed directly to a friend, posted on a bulletin board, etc. When you receive 
+an invitation, you should validate it via the [InvitationIssuer](#invitationissuer). Note that 
+the invitation is a special case of [`Payment`](#payment), and so is associated with a specific [`Issuer`](#issuer).
+
+To participate in a contract instance by making an [offer](#offer), an invitation to that instance must accompany the offer.
+
+An `invitation`'s properties are:
+- The contract's installation in Zoe, including access to its source code.
+- The contract instance this invitation is for.
+- A handle used to refer to this invitation.
+- A description of this invitation's purpose.
+
+## InvitationIssuer
+
+Since [invitations](#invitation) are special cases of [payments](#payment), invitations 
+must have a dedicated [issuer](#issuer), which is the InvitationIssuer.
+
+Zoe has a single `InvitationIssuer` for its entire lifetime. By having a reference to Zoe, 
+a user can get the `InvitationIssuer`. This lets them claim any invitation they 
+receive from someone else by calling `E(invitationIssuer).claim()` with the untrusted
+invitation as the argument. During the claiming process, the invitationIssuer validates
+the invitation. A successful claim also means that invitation is exclusively yours.
```

Comment:
We should mention that depositing in a purse does the same thing, and this is what the wallet does.

Reply:
Added "**Note**: Depositing into an invitation-branded purse also claims an invitation. This is
what the [wallet](#wallet) does."

Reply:
The language above isn't right, but the language used in the document "**Note**: Depositing into an invitation-branded purse also validates an invitation. This is
what the [wallet](#wallet) does." is correct.